### PR TITLE
Added tests for Device service

### PR DESCRIPTION
**Pull request by zanpizmoht**
Signed-off-by: zanpizmoht <zan.pizmoht@comtrade.com>

Test scenarios for adding, deleting and editing Devices are added in this PR.

**Related Issue**
None

**Description of the solution adopted**
Test scenarios that are added in this PR are checking functionalities regarding Devices. All scenarios are added to DeviceServiceI9n.feature. Scenarios are testing if Devices are correctly created, edited and deleted.

**Screenshots**
/

**Additional Notes**
@Coduz This PR has to be merged before PR #2818 which is still waiting for CQ. After this is merged, we will just rebase the #2818 and the CQ will not be needed any more, since the total lines count will be less than 1k.

### CODE REVIEW

**Review by LeoNerdoG — APPROVED** [review]
Hej @coduz,
this PR is second part of device integration tests (first one is #2818) - after this is merged, 2818 will be rabes to latest develop and it will contain less than 1k lines so we will be able to commit this without a CQ. 
Can you please merge this one when travis completes?

**Comment by Coduz** [comment]
Hi @zanpizmoht ,

this now has some conflicts with `develop`.
Can you please fix them?

Regards, 

_Alberto_

**Comment by zanpizmoht** [comment]
Hi @Coduz,
merge conflicts on this branch have been resolved. The branch is now ready to be merged.
Regards, Zan

**Review by Coduz — APPROVED** [review]

**Pull request merged**
